<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayWindow</class>
 <widget class="QMainWindow" name="PlayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RIS的太鼓達人ヽ(´∀`*)ﾉ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_playbg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-100</y>
      <width>1021</width>
      <height>811</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(243, 231, 219);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rrr.qrc">:/images/a9d6394a02957efa11b10b815b763ee4.jpg</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_catpizza">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>500</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/200 (4).gif</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_critical">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>21</width>
      <height>151</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Castellar</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(88, 88, 106);</string>
    </property>
    <property name="text">
     <string>Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_score">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Castellar</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(88, 88, 106);</string>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_play">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>190</y>
      <width>1031</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:0, stop:0.923274 rgba(250, 250, 250, 174), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="resizeAnchor">
     <enum>QGraphicsView::AnchorViewCenter</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>570</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rrr.qrc">:/images/wordball_r.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>570</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rrr.qrc">:/images/wordball_l.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
   <zorder>label_playbg</zorder>
   <zorder>graphicsView_play</zorder>
   <zorder>label_catpizza</zorder>
   <zorder>label_time</zorder>
   <zorder>label_score</zorder>
   <zorder>line_critical</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>遊戲</string>
    </property>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>調整</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>說明</string>
    </property>
    <addaction name="action_howto"/>
    <addaction name="action_ris"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action_pause">
   <property name="text">
    <string>暫停</string>
   </property>
  </action>
  <action name="action_restart">
   <property name="text">
    <string>重新開始</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>我不玩了</string>
   </property>
  </action>
  <action name="action_tostartwindow">
   <property name="text">
    <string>回到開始畫面</string>
   </property>
  </action>
  <action name="action_howto">
   <property name="text">
    <string>要怎麼玩</string>
   </property>
  </action>
  <action name="action_ris">
   <property name="text">
    <string>關於作者萌RIS</string>
   </property>
  </action>
  <action name="action_aboutgame">
   <property name="text">
    <string>關於這ㄍ遊戲</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rrr.qrc"/>
 </resources>
 <connections/>
</ui>
